gid

```diff
@@ -56,11 +55,18 @@ o)  echo ". $OP3"
     ;;
 
 b)  echo ". $OP4"
-    read -p "Enter your branch name (in lower-case-dashed-string): " N
```

Comment:
Introducted `read` variable naming convention to avoid involuntarily overwriting `N`